pages/home/oracle-integrity-staking/slashing-rulebook.mdx

```diff
@@ -0,0 +1,106 @@
+
+### Importance of Data Quality
+
+The Pyth Network is designed to provide reliable and trustworthy financial data to decentralized applications and smart contracts. The quality of the data it provides is paramount for the functioning of these applications and the overall success of the network. Therefore, the role of data publishers and their commitment to data quality cannot be overstated.
+
+## Staking
+
+Staking through OIS is a program to enhance the security of price feeds produced by publisher on the Pyth Network.
+
+Per the design of OIS, a pool aligned with a data publisher accepts stake from the moment such publisher declares its stake account through configuration. The additional stake allows such publisher to earn a delegation fee to contribute to the costs of providing data to the Pyth Network. Data publishers compete with one another to increase the number of price feeds they contribute to, and/or increase the quality of data that allows them to be selected to publish and/or enable higher staking cap for the pool each publisher is aligned to.
+
+## Slashing
+
+In the event of a Pyth data inaccuracy that can be verified against independent data references, a portion of the stake can be reduced from the pools aligned with the publishers responsible for the erroneous data.
+
+### Slashing Conditions
+
+A data quality issue happens when the aggregated price from Pyth is deemed inaccurate. Such inaccuracy has to fulfil the following criteria:
+
+- Prevalent prices for the asset(s) in question and related crosses using the most liquid venues at the time of the incident were at least 250 bps away from the price produced by Pyth
+- The price deviation have lasted for at least 60 seconds
+- Confidence Intervals prints from Pyth do not show an abnormal deviation during the misprint
+- The market conditions are normal and the incident is not related to one or many macro events that make the accurate pricing of assets not feasible
+
+### Slashing Calculation and Distribution
+
+If slashing event confirmed, the Pythian Council will process calculation and distribution of the slashed stake according to the following:
+
+- **Stake Slashed**
+  - capped at 5% of the total amount staked (including the amount delegated) into pools associated with publishers identified as directly responsible for poor data quality. distribution of the slashed amount is uniform amongst publishers and delegator(s)
+  - In the case the total amount staked by the stakers responsible for the data quality issue is nil, no slashing takes place
+- **Loss of Rewards**
+  - the stakers that may be responsible for the data quality issue forfeit any rewards the epoch following the slashing event
+- **Temporary or Permanent Removal:**
+  - Stakers responsible for more than 4 slashing events per calendar year may be excluded permanently from the network exclusion from the set of stakers and/or publishers
```

Comment:
you can't get excluded from the stakers but yes from the publishers and your associated pool will have a soft cap of 0, so you'll be getting 0 rewards

Reply:
this is for the PFC to exclude them as a publisher all together.